<?xml version="1.0" encoding="utf-8" ?>
<dwsync>
<file name="exceptions.py" server="192.168.0.13//root/alligatorproxy/hydrogen/" local="130489395073895315" remote="130489395150000000" Dst="0" />
<file name="__init__.py" server="192.168.0.13//root/alligatorproxy/hydrogen/" local="130485772177106312" remote="130489311220000000" Dst="0" />
<file name="exceptions.py" server="192.168.0.14//root/alligatorproxy/hydrogen/" local="130517943793903737" remote="130517944090000000" Dst="0" />
<file name="service.py" server="192.168.0.14//root/alligatorproxy/hydrogen/" local="130516975978368490" remote="130516976010000000" Dst="0" />
<file name="__init__.py" server="192.168.0.14//root/alligatorproxy/hydrogen/" local="130516759988150137" remote="130516815030000000" Dst="0" />
<file name="exceptions.pyc" server="192.168.0.14//root/alligatorproxy/hydrogen/" local="130517012403975770" remote="130517944180000000" Dst="0" />
<file name="__init__.pyc" server="192.168.0.14//root/alligatorproxy/hydrogen/" local="130517012403955760" remote="130517024020000000" Dst="0" />
</dwsync>